picard/formats/asf.py

```diff
@@ -138,6 +138,7 @@ class ASFFile(File):
         'djmixer': 'WM/DJMixer',
         'mixer': 'WM/Mixer',
         'artists': 'WM/ARTISTS',
+        'albumartists': 'WM/ALBUMARTISTS',
```

Comment:
@lazka  same as artist and artists. artist is a string, artists is a multi valued field.